### DISCUSSION TITLE
Is there a simple way to mark a field as required with Mantine Forms?

### DISCUSSION THREAD

**Discussion by yorkeJohn** [mantinedev/mantine#6549] [Q&A]
Is there a simple way to mark a field as required with Mantine Forms? So far the only way to do it is to add a validation function that checks the input's length, which seems a little over-complicated.

e.g.
```ts
  const form = useForm<FormData>({
    mode: 'uncontrolled',
    validate: {
      name: value => (!value?.length ? 'Please enter a project name' : null),
      summary: value => (!value?.length ? 'Please enter a project summary' : null)
    }
  })
```

**Comment by Sajarin-M** [comment]
Mantine forms have some built-in validators like `isNotEmpty` for that.
Refer [this section](https://mantine.dev/form/validators/) in docs
